app/internal/page/certificate_provider_address.go

```diff
@@ -64,7 +65,13 @@ func CertificateProviderAddress(logger Logger, tmpl template.Template, addressCl
 				addresses, err := addressClient.LookupPostcode(r.Context(), data.Form.LookupPostcode)
 				if err != nil {
 					logger.Print(err)
-					data.Errors.Add("lookup-postcode", validation.CustomError{Label: "couldNotLookupPostcode"})
+
+					if errors.As(err, &place.InvalidPostcodeError{}) {
+						data.Errors.Add("lookup-postcode", validation.EnterError{Label: "invalidPostcode"})
```

Comment:
On https://design-system.service.gov.uk/patterns/addresses/ they use "Enter a real postcode" which might be nicer than the word "valid"?

Reply:
Good question but we've deliberately chosen to go with **valid** over **real** based on conversations around this in the content community. ONS and other design systems also use valid over real: 

From ONS:  If the postcode entered is not valid
Use “Enter a valid UK postcode”. 

See https://ons-design-system.netlify.app/patterns/addresses/#manual-addresses 

"Real" to the user is not helpful if they are entering a postcode they believe to be real. I think it sounds like we are casting doubt on the user's input. Whereas 'valid' seems a bit more neutral. Other teams have also had this discussion and ended up going with 'valid'. Can send Slack threads about this, if interested.